editor/src/components/canvas/canvas-strategies/strategies/absolute-duplicate-strategy.tsx

```diff
@@ -76,36 +78,59 @@ export function absoluteDuplicateStrategy(
           ...customStrategyState.duplicatedElementNewUids,
         }
         let duplicateCommands: Array<CanvasCommand> = []
-        let newPaths: Array<ElementPath> = []
 
         flattenedSelectionForMultiSelect.forEach((selectedElement) => {
           const selectedElementString = EP.toString(selectedElement)
           const newUid =
             duplicatedElementNewUids[selectedElementString] ??
             generateUidWithExistingComponents(canvasState.projectContents)
-          duplicatedElementNewUids[selectedElementString] = newUid
-
-          const newPath = EP.appendToPath(EP.parentPath(selectedElement), newUid)
 
-          newPaths.push(newPath)
+          duplicatedElementNewUids[selectedElementString] = newUid
           duplicateCommands.push(duplicateElement('always', selectedElement, newUid, 'before'))
         })
 
         return strategyApplicationResult(
           [
             ...duplicateCommands,
-            setElementsToRerenderCommand([...selectedElements, ...newPaths]),
-            updateSelectedViews('always', selectedElements),
-            updateFunctionCommand('always', (editorState, commandLifecycle) =>
-              runMoveStrategy(
-                canvasState,
-                editorState,
+            updateFunctionCommand('always', (state, commandLifecycle) => {
+              const selectedUids = selectedElements.map(EP.toUid)
```

Comment:
instead of this whole function body, why is it not enough to do something along these lines?

```typescript
const newSelectedViews = selectedViews.map(selectedView => reparentedToPaths[EP.toString(selectedView)])
...
setElementsToRerenderCommand(newSelectedViews)
updateSelectedViews(newSelectedViews)

```